presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -315,6 +316,12 @@ public static boolean isHudiRealtimeSplit(Map<String, String> customSplitInfo)
         return HoodieRealtimeFileSplit.class.getName().equals(customSplitClass);
     }
 
+    public static boolean isBootstrapBaseFileSplit(Map<String, String> customSplitInfo)
```

Comment:
See my above comment. This method is probably not needed.

Reply:
Removed.